sympy/physics/mechanics/tests/test_pathway.py

```diff
@@ -0,0 +1,143 @@
+"""Tests for the ``sympy.physics.mechanics._pathway.py`` module."""
+
+from __future__ import annotations
+
+from typing import Any, Sequence
+
+import pytest
+
+from sympy.core.symbol import Symbol
+from sympy.functions.elementary.miscellaneous import sqrt
+from sympy.physics.mechanics import Point, ReferenceFrame, dynamicsymbols
+from sympy.physics.mechanics._pathway import LinearPathway
+
+
+class TestLinearPathway:
+
+    @staticmethod
+    @pytest.mark.parametrize(
+        'args, kwargs',
+        [
+            (((Point('pO'), Point('pI')), ), {}),
+            (([Point('pO'), Point('pI')], ), {}),
+            ((), {'attachments': (Point('pO'), Point('pI'))}),
+            ((), {'attachments': [Point('pO'), Point('pI')]}),
```

Comment:
What is the reason for having the same parametrization twice?

Reply:
One's a list, one's a tuple. Just checking that the common sequence types are allowed.